src/ci/docker/host-x86_64/mingw-check/Dockerfile

```diff
@@ -37,4 +38,5 @@ ENV SCRIPT python3 ../x.py --stage 2 test src/tools/expand-yaml-anchors && \
            python3 ../x.py doc --stage 0 library/std && \
            /scripts/validate-toolstate.sh && \
            # Runs checks to ensure that there are no ES5 issues in our JS code.
-           es-check es5 ../src/librustdoc/html/static/*.js
+           es-check es5 ../src/librustdoc/html/static/*.js && \
+	   eslint ../src/librustdoc/html/static/*.js
```

Comment:
Wouldn't it be better to make this part of x.py? That way it's easier for people to run locally,  this is really hard to find.

Reply:
Well, installing eslint isn't "easy" (requires to have `npm` then to run `npm install eslint` and if you're lucky, everything has been installed correctly) and I have no idea how all of that can work on windows. Considering that it fails pretty quickly and everyone has access to the log, I think it's enough.

Reply:
Also I juste realized that I added a tab instead of whitespaces...